templates/repo/view_content.tmpl

```diff
@@ -41,8 +41,9 @@
 		<a href="{{.Repository.Link}}/find/{{.RefTypeNameSubURL}}" class="ui compact basic button">{{ctx.Locale.Tr "repo.find_file.go_to_file"}}</a>
 	{{end}}
 
-	{{if and .CanWriteCode .RefFullName.IsBranch (not .Repository.IsMirror) (not .Repository.IsArchived) (not .IsViewFile)}}
-		<button class="ui dropdown basic compact jump button"{{if not .Repository.CanEnableEditor}} disabled{{end}}>
+	{{if and .RefFullName.IsBranch (not .IsViewFile) .Repository.CanEnableEditor}}
+		<button class="ui dropdown basic compact jump button"
```

Comment:
IIRC disabled button is not able to show tooltip

Reply:
Indeed, it seems to not show up. I'll just hide the button again when not signed in then.